redi-connect-front/src/pages/app/me/Me.scss

```diff
@@ -6,4 +7,11 @@
     margin-bottom: 1.5rem;
     border-bottom: 1px solid $grey-light;
   }
+
+  &__block--mobile {
```

Comment:
Use either a block or a modifier. Don't bind the two together.

Reply:
done